README.asciidoc

```diff
@@ -1,122 +1,111 @@
+==== 5 minutes tests
 
-===== Vagrant
+===== Demo site
 
-Just want to test Rudder without worrying about a full installation?  We provide a https://github.com/Normation/rudder-vagrant[Vagrant set-up]. See https://www.rudder-project.org/site/get-rudder/vagrant/[Rudder Vagrant page] for a getting-started with Rudder on Vagrant.
+You can see a demo version of Rudder: http://demo.rudder-project.org/
 
+===== Vagrant
 
-=== Rudder components and projects overview
+Just want to test Rudder without worrying about a full installation, but with real
+virtual machines so that you can actually configure and test things, contrary to
+the demo? We provide a https://github.com/Normation/rudder-vagrant[Vagrant set-up].
+See https://www.rudder-project.org/site/get-rudder/vagrant/[Rudder Vagrant page]
+for a getting-started with Rudder on Vagrant.
 
-Rudder is now composed of several projects:
+===== Rudder Test Framework (rtf)
 
-|====
-|Component | Description and GitHub Projects
+Rudder test framework is a tool that allows to define full environnements and set
+them up for you. For example, you can define a plateform with a Rudder server on
+Debian stable, 2 nodes on CentOS connected to it, and a Rudder Relay Server with
+its own 2 Debian nodes with 3 lines of configuration and command.
 
-| Rudder Techniques | 
-Provided Techniques coming in the base set-up of Rudder 
-https://github.com/Normation/rudder-techniques
+See https://github.com/Normation/rudder-tests for more information.
 
-| Rudder packages | 
-This project contains all the logic to build both server and agent packages for Rudder
-https://github.com/Normation/rudder-packages
+==== Installing on real environments
 
-| Rudder server | 
-This is the Scala web application It is composed of several Scala projects: 
-https://github.com/Normation/rudder-parent-pom
-https://github.com/Normation/rudder-commons
-https://github.com/Normation/scala-ldap
-https://github.com/Normation/cf-clerk
-https://github.com/Normation/rudder
+===== Linux server packages
 
-| Rudder inventory | 
-This part is in charge of parsing fusion inventory reports for Rudder
-https://github.com/Normation/ldap-inventory
+The easiest way to test and use Rudder for prototyping or production is to install
+it from the provided Linux packages.
+For the server, the main current .rpm-based (RedHat, CentOS, SLES) and .deb-based
+(Debian, Ubuntu) distribution are supported. See https://www.rudder-project.org/site/get-rudder/downloads/
+for details about repository and available versions.
 
-| Rudder agent | 
-This project contains the CLI for Rudder agent
-https://github.com/Normation/rudder-agent
+===== Rudder agent packages
 
-| ncf framework | 
-A powerful and structured CFEngine framework used to build Rudder Techniques
-https://github.com/Normation/ncf
-
-| Rudder tools | 
-Nice tools around Rudder 
-https://github.com/Normation/rudder-tools
+Rudder agent is extremelly versatil and packages are available for the same
```

Comment:
the same on many more versions => a wide set of distributions and releases
(or something like his)